lastplot/graph_constructor.py

```diff
@@ -1,41 +1,72 @@
+import matplotlib.pyplot as plt
```

Comment:
why did you import it??????????????????????????????????????????????????????????????????????????????????????????????
```suggestion
```